src/dispatch/plugins/dispatch_slack/views.py

```diff
@@ -756,7 +763,7 @@ def verify_timestamp(timestamp: int):
 
     # Dispatch action functions to be executed in the background
     for f in action_functions(action_name):
-        background_tasks.add_task(f, user_email, incident_id, action)
+        background_tasks.add_task(f, user_id, user_email, incident_id, action)
```

Comment:
Should `user_id` be a kwarg? It is not used in a lot of functions that are added as background tasks (e.g. `new_status_report_flow()`).

Reply:
It's not optional for the functions that do use it, so I don't think a kwarg makes sense.